docs/developer-guide/plugins-scenarios.md

```diff
@@ -216,15 +215,14 @@ and possibly running out of memory. Styx only accumulates up to `maxContentBytes
 Streaming HTTP body content can be transformed both synchronously and asynchronously.
 However there are some pitfalls you need to know:
 
- - Reference counting. Styx exposes the content stream as an observable of reference counted
-   Netty `ByteBuf` objects. Ensure the reference counts are correctly decremented when
-   buffers are transformed. 
-   
- - Continuity (or discontinuity) of Styx content observable. Each content tranformation with 
-  `map` or `flatMap` is a composition of some source observable. So is each content transformation 
+ - Reference counting. Styx exposes the content stream as a `ByteStream` of reference counted
+  `Buffer` objects. 
+  
+ - Continuity (or discontinuity) of Styx content `ByteStream`. Each content transformation with 
```

Comment:
"Continuity (or discontinuity) of Styx content `ByteStream`. Each content" ->
"**Continuity (or discontinuity) of the content `ByteStream`**: each content"

Reply:
👍